nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml

```
@@ -89,14 +109,14 @@ properties:
           - tax_id
       number:
         type: string
-        description: The sender's ID number. Mandatory when sending cross-border transactions from AU
+        description: The sender's identification number
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's country of issue. Mandatory when sending cross-border transactions from AU
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the sender's identification
         minLength: 2
         maxLength: 2
       date_of_expiry:
         type: string
-        description: The sender's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`
```

```suggestion
        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`.
```

To be consistent, should it be either `YYYY-MM-DD` or `yyyy-mm-dd`?

---

updated.